src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.DataContracts/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+namespace MyExtensionsApp.DataContracts;
+
+/// <summary>
+/// A Weather Forecast for a specific date
+/// </summary>
+/// <param name="Date">Gets the Date of the Forecast.</param>
+/// <param name="TemperatureC">Gets the Forecast Temperature in Celsius.</param>
+/// <param name="Summary">Get a description of how the weather will feel.</param>
+public record WeatherForecast(DateTime Date, int TemperatureC, string? Summary)
+{
+	/// <summary>
+	/// Gets the Forecast Temperature in Fahrenheit
+	/// </summary>
+	public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
```

Comment:
I'm not sure why this uses a static constant when this should be 5/9

Reply:
The implementation was pulled from the Uno template and the .NET Team's API template and just updated for your love of records

Reply:
Just because MS shipped something that's not correct...... let's fix this please